src/pose_solver/pose_solver.py

```diff
@@ -664,14 +628,14 @@ def _estimate_target_pose_from_ray_set_and_single_marker_id(
         point_bottom_right = numpy.array(object_points[2])
         point_bottom_left = numpy.array(object_points[3])
         marker_side = numpy.linalg.norm(point_top_right - point_top_left)
-        if marker_side <= EPSILON:
+        if marker_side <= PoseSolverParameters.EPSILON:
             raise RuntimeError("Input marker points must define a square of side length larger than zero.")
         marker_diag = numpy.sqrt(numpy.square(marker_side) * 2)
-        if abs(numpy.linalg.norm(point_bottom_right - point_top_right) - marker_side) > EPSILON or \
-           abs(numpy.linalg.norm(point_bottom_left - point_bottom_right) - marker_side) > EPSILON or \
-           abs(numpy.linalg.norm(point_top_left - point_bottom_left) - marker_side) > EPSILON or \
-           abs(numpy.linalg.norm(point_top_left - point_bottom_right) - marker_diag) > EPSILON or \
-           abs(numpy.linalg.norm(point_top_right - point_bottom_left) - marker_diag) > EPSILON:
+        if abs(numpy.linalg.norm(point_bottom_right - point_top_right) - marker_side) > self._parameters.EPSILON or \
+           abs(numpy.linalg.norm(point_bottom_left - point_bottom_right) - marker_side) > self._parameters.EPSILON or \
+           abs(numpy.linalg.norm(point_top_left - point_bottom_left) - marker_side) > self._parameters.EPSILON or \
+           abs(numpy.linalg.norm(point_top_left - point_bottom_right) - marker_diag) > self._parameters.EPSILON or \
+           abs(numpy.linalg.norm(point_top_right - point_bottom_left) - marker_diag) > self._parameters.EPSILON:
```

Comment:
Have you checked if this works? what I see in the data structure is that this is a `Field`, and not a constant.

You can just leave `EPSILON` as a constant in this file.